flax/linen/sequential.py

```diff
@@ -0,0 +1,13 @@
+from typing import Sequence
+from flax.linen.module import Module
+from typing import Type
+
+
+
+class Sequential(Module):
```

Comment:
Please add documentation to both the Module and the `__call__` function. For instance as we have for the `Dense` layer: https://github.com/google/flax/blob/master/flax/linen/linear.py#L140